dependencies.go

```diff
@@ -99,3 +102,30 @@ func (f FastlaneRunner) InstallDependencies(opts EnsureDependenciesOpts) error {
 
 	return nil
 }
+
+func (f FastlaneRunner) reportRubyVersion(useBundler bool, bundlerVersion string) {
+	var versionCmd command.Command
+	if useBundler {
+		versionCmd = f.rbyFactory.CreateBundleExec("ruby", []string{"--version"}, bundlerVersion, nil)
```

Comment:
If rbenv is used it may select a different ruby version based on `.ruby-version` file for example, this may be different from the one reported by `ruby version` I think.